src/universal/components/Dashboard/DashLayout.js

```diff
@@ -0,0 +1,64 @@
+import React, {PropTypes} from 'react';
+import look, {StyleSheet} from 'react-look';
+
+import Helmet from 'react-helmet';
+import {head} from 'universal/utils/clientOptions';
+
+import NotificationBar from 'universal/components/NotificationBar/NotificationBar';
+
+let styles = {};
+
+const DashLayout = (props) => {
```

Comment:
Scratch that. They _always_ fail with async functions if any `arrow-parens` rule is used. Ugh. I'm punting on the rule for now.

Let's implicitly use always. Fix em as you see 'em.